index.bs

```diff
@@ -1523,14 +1523,7 @@ NOTE: This diagram demonstrates how two potential controllers would be exposed w
 Layers {#layers}
 ======
 
-XRLayer {#xrlayer-interface}
--------
-
-An {{XRLayer}} defines a source of bitmap images and a description of how the image is to be rendered to the [=/XR device=]. Initially only one type of layer, the {{XRWebGLLayer}}, is defined but future revisions of the spec may extend the available layer types.
-
-<pre class="idl">
-[SecureContext, Exposed=Window] interface XRLayer {};
-</pre>
+NOTE: While this specification only defines the {{XRWebGLLayer}} layer, future modules are expected to add additional layer types and the image sources that they draw from.
```

Comment:
> modules

Teeeeechnically we haven't officially decided this is the plan... Perhaps let's use a less specific phrase for the moment :)